hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/UpgradeConfig.java

```diff
@@ -24,4 +25,5 @@ public record UpgradeConfig(
         @ConfigProperty(
                         value = "artifacts.path",
                         defaultValue = "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade/current")
+                @NetworkProperty
```

Comment:
This appears to be a file path, which would be node-specific.

Reply:
Updated.